src/components/registration/registrationForm/registrationForm.tsx

```diff
@@ -70,6 +70,8 @@ const validationSchema = Yup.object().shape({
 
 export const RegistrationForm = (): JSX.Element => {
   const dispatch = useAppDispatch();
+  // eslint-disable-next-line
+  const [open, setOpen] = useState(false);
```

Comment:
try to make like this to avoid eslint errors:

`const [open, setOpen] = useState<boolean>(false);`